src/pjstadig/util.cljc

```diff
@@ -83,11 +85,32 @@
                         (.clear sb))
                       (print-expected actual)))))))
 
+(defn- convert-event [{:keys [actual expected] :as event}]
+  (let [diffs (when (and (seq? actual)
+                         (seq actual)
+                         (= 'not (first actual))
+                         (seq? (second actual))
+                         (seq (second actual))
+                         (#{'clojure.core/= '=} (first (second actual)))
+                         (< 2 (count (second actual))))
+                (let [a (nth (second actual) 1)
+                      more (drop 2 (second actual))]
+                  (map vector
+                       more
+                       (map #(take 2 (m/diff a %)) more))))
+        expected (if (seq diffs)
+                   (nth (second actual) 1)
+                   expected)]
+    (assoc event
+           :diffs diffs
+           :expected expected)))
+
+
 (defn define-fail-report []
-  #?(:clj (defmethod report :fail [& args]
```

Comment:
@pjstadig would you then be ok if this PR could be brought in with the `#?(clj ` `defmethod` untouched and the `#?(cljs` part taking the new code path - no BONUS FEATURE?